pkg/oc/admin/diagnostics/diagnostics/cluster/aggregated_logging/diagnostic.go

```diff
@@ -207,51 +234,3 @@ and updating the annotation:
   'openshift.io/node-selector' : ""
 
 `
-
-func retrieveLoggingProject(r types.DiagnosticResult, loggingURL string, projectClient projecttypedclient.ProjectsGetter, routeClient routetypedclient.RoutesGetter) string {
-	r.Debug("AGL0010", fmt.Sprintf("masterConfig.AssetConfig.LoggingPublicURL: '%s'", loggingURL))
-	projectName := ""
-	if len(loggingURL) == 0 {
-		r.Debug("AGL0017", "masterConfig.AssetConfig.LoggingPublicURL is empty")
-		return projectName
-	}
-
-	loggingUrl, err := url.Parse(loggingURL)
-	if err != nil {
-		r.Error("AGL0011", err, fmt.Sprintf("Unable to parse the loggingPublicURL from the masterConfig '%s'", loggingURL))
-		return projectName
-	}
-
-	routeList, err := routeClient.Routes(metav1.NamespaceAll).List(metav1.ListOptions{LabelSelector: loggingSelector.AsSelector().String()})
-	if err != nil {
-		r.Error("AGL0012", err, fmt.Sprintf("There was an error while trying to find the route associated with '%s' which is probably transient: %s", loggingUrl, err))
-		return projectName
-	}
-
-	for _, route := range routeList.Items {
-		r.Debug("AGL0013", fmt.Sprintf("Comparing URL to route.Spec.Host: %s", route.Spec.Host))
-		if loggingUrl.Host == route.Spec.Host {
-			if len(projectName) == 0 {
-				projectName = route.ObjectMeta.Namespace
-				r.Info("AGL0015", fmt.Sprintf("Found route '%s' matching logging URL '%s' in project: '%s'", route.ObjectMeta.Name, loggingUrl.Host, projectName))
-			} else {
-				r.Warn("AGL0019", nil, fmt.Sprintf("Found additional route '%s' matching logging URL '%s' in project: '%s'.  This could mean you have multiple logging deployments.", route.ObjectMeta.Name, loggingUrl.Host, projectName))
-			}
-		}
-	}
-	if len(projectName) == 0 {
-		message := fmt.Sprintf("Unable to find a route matching the loggingPublicURL defined in the master config '%s'. Check that the URL is correct and aggregated logging is deployed.", loggingUrl)
-		r.Error("AGL0014", errors.New(message), message)
-		return ""
-	}
-	project, err := projectClient.Projects().Get(projectName, metav1.GetOptions{})
-	if err != nil {
-		r.Error("AGL0018", err, fmt.Sprintf("There was an error retrieving project '%s' which is most likely a transient error: %s", projectName, err))
-		return ""
-	}
-	nodeSelector, ok := project.ObjectMeta.Annotations["openshift.io/node-selector"]
-	if !ok || len(nodeSelector) != 0 {
-		r.Warn("AGL0030", nil, fmt.Sprintf(projectNodeSelectorWarning, projectName))
```

Comment:
We lose this warning on the project check which may be significant to restrict the logging project to infra nodes

Reply:
good catch. Added that back in as well as a test whether the project with such name even exists (and report early error if it doesn't)